client/state/action-types.js

```diff
@@ -178,6 +178,7 @@ export const HAPPYCHAT_SET_MESSAGE = 'HAPPYCHAT_SET_MESSAGE';
 export const HAPPYCHAT_TRANSCRIPT_RECEIVE = 'HAPPYCHAT_TRANSCRIPT_RECEIVE';
 export const HAPPYCHAT_TRANSCRIPT_REQUEST = 'HAPPYCHAT_TRANSCRIPT_REQUEST';
 export const HELP_COURSES_RECEIVE = 'HELP_COURSES_RECEIVE';
+export const HELP_SELECTED_SITE = 'HELP_SELECTED_SITE';
```

Comment:
Pinging again on this — I still think a more action-y name like ending with `SITE_SELECT` is clearer than `SELECTED_SITE` ("selected" is both a past tense verb and an adjective).